src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -73,6 +72,20 @@ impl Default for TimeTriggerInterval {
     }
 }
 
+#[derive(Debug, Error)]
+enum TimeTrigerIntervalError {
+    #[error("The 'Seconds' value specified as a time trigger is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Second(TimeTriggerInterval),
+    #[error("The 'Minutes' value specified as a time trigger is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Minute(TimeTriggerInterval),
+    #[error("The 'Hours' value specified as a time trigger is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Hour(TimeTriggerInterval),
+    #[error("The 'Days' value specified as a time trigger is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Day(TimeTriggerInterval),
+    #[error("The 'Weeks' value specified as a time trigger is out of bounds, ensure it fits within an i64: : '{0:?}'")]
+    Week(TimeTriggerInterval),
+}
+
```

Comment:
I dont think it's necessary either. For users, all these parameters are from the item `interval`, so just "the interval value....." is enough imo.

Reply:
I added a comment that was something to the effect of "we should add context to errors". I obviously didnt add enough context, sorry. My point was this: if a user has a huge config with many intervals we need to set up the error handling such that when surfaced to the user, they know _which_ interval was out of bounds.

So we need to: 
1. Simply add a position descriminant to this error:
```
```

Reply:
I don't think we can access the name anywhere close to the triggers. They're owned in the config module but are not available anywhere within the append module that I can find

Reply:
Alright I've been staring at this and tried a slew of different things, none of which I think get close to what you're asking for here. I think I need some assistance.

Like I mentioned we don't have access to the appender name, and I'm not really sure what you mean by a Position discriminant. The Error already reports the type of the interval (Seconds, Minutes, Hours, etc.) and I'm not sure what else we can add of value to the user.

Reply:
I dont think it's necessary to include the appender name. The incorrect interval numbers is enougn to assist users to locate the configuration. Even if they have duplicate incorrect number, they need fix them all because all these numbers will be incorrect for  `interval`